osu.Game/Screens/OnlinePlay/Multiplayer/Multiplayer.cs

```diff
@@ -57,7 +57,7 @@ protected override void UpdatePollingRate(bool isIdle)
         protected override Room CreateNewRoom() =>
             new Room
             {
-                Name = { Value = $"{API.LocalUser}'s awesome room" },
+                Name = { Value = $"{API.LocalUser}{(API.LocalUser.ToString().ToLower().EndsWith("s") ? "'" : "'s")} awesome room" },
```

Comment:
This won't work when this string gets localized.
Don't know how we'll handle possessive forms in localization.